src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -498,6 +499,18 @@ public List<String> onTabComplete(CommandSender sender, Command command, String
 						default:
 							return Collections.emptyList();
 					}
+				case "trusttown":
+					switch (args.length) {
+						case 2:
+							return NameUtil.filterByStart(Arrays.asList("add", "remove", "list"), args[1]);
+						case 3:
+							if (args[1].equalsIgnoreCase("add") || args[1].equalsIgnoreCase("remove"))
+								return getTownyStartingWith(args[2], "t");
+							else
+								return Collections.emptyList();

```

Comment:
```suggestion
							return Collections.emptyList();
```
remove redundant else: 'else' branch may be unwrapped, as the 'if' branch never completes normally

Reply:
Implemented as suggested.